sdk/aqueduct/integrations/mongodb_integration.py

```diff
@@ -57,6 +58,7 @@ def find(
                 Whether to run this operator lazily. See https://docs.aqueducthq.com/operators/lazy-vs.-eager-execution .
         """
         op_name = name or self._dag.get_unclaimed_op_name(prefix="%s query" % self.name())
+        artf_name = output or artifact_name_from_op_name(op_name)
```

Comment:
We should check whether the artifact name is available:
```
        artifact_name = output or artifact_name_from_op_name(op_name)
        existing = self._dag.get_artifact_by_name(artifact_name)
        if existing is not None:
            raise InvalidUserActionException(
                "Artifact with name `%s` has already been created locally. Artifact names must be unique."
                % name,
                )
```

Reply:
Actually it's actually easier for me to just do it all in https://github.com/aqueducthq/aqueduct/pull/953 since I'm writing a helper for this anyways.

Reply:
It's done! https://github.com/aqueducthq/aqueduct/pull/953

Reply:
yes! Thank you!